libs/payments/customer/src/lib/util/getOfferingAndInterval.spec.ts

```diff
@@ -0,0 +1,32 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// TODO
+
+import {
+  StripePriceFactory,
+  StripePriceRecurringFactory,
+} from '@fxa/payments/stripe';
+import { SubplatInterval } from '../types';
+import { doesPriceMatchSubplatInterval } from './doesPriceMatchSubplatInterval';
+
+describe('doesPriceMatchSubplatInterval', () => {
```

Comment:
This spec is for `getOfferingAndInterval` 👀

Reply:
Ah right, that was the TODO! Sorry about that.